Makefile

```diff
@@ -266,9 +269,22 @@ admin.kubeconfig:
 aks.kubeconfig:
 	hack/get-admin-aks-kubeconfig.sh
 
+.PHONY: go-tidy
```

Comment:
Why each individual PHONY and not added to the overall PHONY at the bottom of the file?

Reply:
Do you mean why we need to create three targets (with one PHONY per target) instead of one target (and one PHONY)?

Each go module command has a different purpose, and by separating them, I believe, we get better code readability and flow of the targets. They could be grouped together as they are used today, but separation empowers more flexibility :) Having said, I don't have a strong opinion on that.